deny.toml

```diff
@@ -202,5 +202,6 @@ allow-registry = ["https://github.com/rust-lang/crates.io-index"]
 # List of URLs for allowed Git repositories
 allow-git = [
     "https://github.com/radicle-dev/radicle-surf",
```

Comment:
Actually, the other two here can be removed :sweat_smile:

Reply:
Oh!